debug is now a keyword taken by meson >= 0.48.0 and will error out if used as an option

diff --git a/meson.build b/meson.build
index 7b0739f..98a73a1 100644
--- meson.build
+++ meson.build
@@ -929,7 +929,7 @@ conf.set_quoted('GETTEXT_PACKAGE', meson.project_name())
 # substs.set('DEBUGTTY', get_option('debug-tty'))
 #endif // 0
 
-debug = get_option('debug')
+debug = get_option('debug-extra')
 enable_debug_hashmap = false
 enable_debug_mmap_cache = false
 #if 1 /// additional elogind debug mode
diff --git a/meson_options.txt b/meson_options.txt
index 84100b6..f13f904 100644
--- meson_options.txt
+++ meson_options.txt
@@ -82,7 +82,7 @@ option('kexec-path', type : 'string', description : 'path to kexec')
 # option('memory-accounting-default', type : 'boolean',
 #        description : 'enable MemoryAccounting= by default')
 #else
-option('debug', type : 'string',
+option('debug-extra', type : 'string',
        description : 'enable extra debugging (elogind,hashmap,mmap-cache)')
 #endif // 0
 

